app/views/govuk_publishing_components/components/_big_number.html.erb

```diff
@@ -0,0 +1,34 @@
+<%
+  href ||= nil
+  number ||= nil
+  label ||= nil
+  href ||= nil
+  data_attributes ||= nil
+  classes = ["gem-c-big-number__value"]
+  
+  if label.nil? && href
+    classes << "gem-c-big-number__value--decorated"
+  end
+%>
+<% big_number_value = capture do %>
+  <%= tag.span class: classes do %>
+    <%= number %>
+  <% end %>
+
+  <% unless label.nil? %>
+    <% # add a virtual space here to handle screen readers printing dictations without a space between the number and the label %>
+    <span class="govuk-visually-hidden">&nbsp;</span>
+    <span class="gem-c-big-number__label">
+      <%= label %>
+    </span>
+  <% end %>
+<% end %>
+<% if number %>
```

Comment:
This checks whether or not to render the component, correct? Would it make more sense for it to wrap the whole of the code (i.e. from line 13 onwards) rather than just this part?